src/app/interfaces/alert.interface.ts

```diff
@@ -3,7 +3,7 @@ import { AlertPolicy } from 'app/enums/alert-policy.enum';
 import { ApiTimestamp } from 'app/interfaces/api-date.interface';
 
 export interface Alert {
-  args: string;
+  args: string | { device: string; message: string };
```

Comment:
This is too specific as arguments can be all sort of things. Can we do `unknown` and type where it's being used?